x/bank/app_test.go

```diff
@@ -114,14 +114,14 @@ func TestMsgSendWithAccounts(t *testing.T) {
 	require.Equal(t, acc, res1.(*auth.BaseAccount))
 
 	// Run a CheckDeliver
-	mock.SignCheckDeliver(t, mapp.BaseApp, []sdk.Msg{sendMsg1}, []int64{0}, []int64{0}, true, priv1)
+	mock.CheckAndDeliverGenTx(t, mapp.BaseApp, []sdk.Msg{sendMsg1}, []int64{0}, []int64{0}, true, priv1)
```

Comment:
Can you explain the rename? Feels a bit awkward. If anything the right sequence would be like `GenSignCheckDeliverTx` since we generate a tx, sign it, run CheckTx, then DeliverTx

Also note there will be some serious merge conflicts with https://github.com/cosmos/cosmos-sdk/pull/1172